### ISSUE TITLE
Use the same font for the tracking information on the shipped email

### ISSUE THREAD

**Issue by jibees** [openfoodfoundation/openfoodnetwork#8698]
## Description

_Originally posted by @drummer83 in https://github.com/openfoodfoundation/openfoodnetwork/issues/8541#issuecomment-1007604058_

## Expected Behavior
The tracking information should use the same font that the email body. 

## Actual Behaviour
The tracking information use another font that the email body.

## Steps to Reproduce
Test the shipped email

## Animated Gif/Screenshot
![](https://user-images.githubusercontent.com/1790176/148584133-2144aa78-0be5-4aa1-a153-917d1f823cc4.png)

## Severity

bug-s4: it's annoying, but you can use it

**Comment by RachL** [comment]
@jibees is this a good first issue?

**Comment by jibees** [comment]
@RachL As we linked the _probably need to be modified_ file in the issue, I think too ;)

**Comment by RachL** [comment]
@jibees cool! Which column in https://github.com/orgs/openfoodfoundation/projects/2 do you think?

**Comment by jibees** [comment]
@RachL I'd say `Backend Easy`, but didn't manage to add to this column ;)

**Comment by mgrigoriev8109** [comment]
Just to clarify the change that's being requested in this issue. Currently in the shipped email the tracking information uses the Helvetica Neue font, the same font that's used for every other <p> tag except for the shipment details themselves. 
I've attached a picture showing what I mean by this: If I increase the font size of tracking information to 17px, you can see it's identical to everything apart from `Garlic 1g`. 

Is this issue requesting that the `Garlic 1g`, in other words the items inside of @shipment.manifest.each change to be the same font as the rest of the email?

![Screenshot from 2022-05-26 16-54-33](https://user-images.githubusercontent.com/43343880/170578947-68bf99f9-c4c0-47dc-9604-3d48e8261f74.png)

In other words is this the desired stylistic change?

![Screenshot from 2022-05-26 16-59-56](https://user-images.githubusercontent.com/43343880/170579466-b0ace142-4853-4365-bb50-e90c24fd120a.png)

**Comment by drummer83** [comment]
Thanks for working on this, @mgrigoriev8109!

This issue is about the tracking information only. Increasing the size as you proposed would be great!

I am not sure about the "garlic". This is out of scope here and maybe from a design perspective it would be desirable to keep a different font there to highlight it. This needs discussion with our product team, @openfoodfoundation/train-drivers-product-owners .

My proposal is: Provide a pull request changing the tracking information only and if you like (and maybe have a design talent) have a look at wishlist #71 and let's define next steps for the email improvements there.

What do you think?

**Comment by mgrigoriev8109** [comment]
Thank you so much for the feedback @drummer83 ! I'm still getting used to GitHub, but I'm guessing it would be more appropriate for me to continue the conversation regarding the wishlist in that particular issue - so I'll head over there :)
